consul/tests/conftest.py

```diff
@@ -0,0 +1,74 @@
+# (C) Datadog, Inc. 2010-2017
+# All rights reserved
+# Licensed under Simplified BSD License (see LICENSE)
+import subprocess
+import os
+import logging
+
+import pytest
+import requests
+
+import common
+
+log = logging.getLogger(__file__)
+
+
+def wait_for_cluster():
+    """
+    Wait for the slave to connect to the master
+    """
+    from time import sleep
+    for i in xrange(0, 20):
+        sleep(1)
+        try:
+            res = requests.get("{}/v1/status/peers".format(common.URL))
+            # Wait for all 3 agents to join the cluster
+            if len(res.json()) == 3:
+                return True
+        except Exception as e:
+            log.info("Error connecting to the cluster: %s", e)
+            pass
+
+    return False
+
+
+@pytest.fixture(scope="session")
+def spin_up_consul():
```

Comment:
nit: being a fixture it's better to use a declarative naming than an imperative one, I'd use something like `def consul_cluster()`